### ISSUE TITLE
Error while running ipython console --kernel java

### ISSUE THREAD

**Issue by khayyatzy** [Bachmann1234/java9_kernel#19]
Hi,

I get the following error while trying to run command: ipython console --kernel java

Any Idea how to solve this? I am using MAC OS to run this command

 ipython console --kernel java
[TerminalIPythonApp] WARNING | Subcommand `ipython console` is deprecated and will be removed in future versions.
[TerminalIPythonApp] WARNING | You likely want to use `jupyter console`... continue in 5 sec. Press Ctrl-C to quit now.
Jupyter Console 4.1.1

Traceback (most recent call last):
  File "//anaconda/lib/python3.5/site-packages/pexpect/expect.py", line 97, in expect_loop
    incoming = spawn.read_nonblocking(spawn.maxread, timeout)
  File "//anaconda/lib/python3.5/site-packages/pexpect/pty_spawn.py", line 452, in read_nonblocking
    raise TIMEOUT('Timeout exceeded.')
pexpect.exceptions.TIMEOUT: Timeout exceeded.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "//anaconda/lib/python3.5/runpy.py", line 170, in _run_module_as_main
    "__main__", mod_spec)
  File "//anaconda/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/refops/bin/java9_kernel-master/javakernel/__main__.py", line 9, in <module>
    IPKernelApp.launch_instance(kernel_class=JavaKernel)
  File "//anaconda/lib/python3.5/site-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-122>", line 2, in initialize
  File "//anaconda/lib/python3.5/site-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, _args, *_kwargs)
  File "//anaconda/lib/python3.5/site-packages/ipykernel/kernelapp.py", line 421, in initialize
    self.init_kernel()
  File "//anaconda/lib/python3.5/site-packages/ipykernel/kernelapp.py", line 360, in init_kernel
    user_ns=self.user_ns,
  File "//anaconda/lib/python3.5/site-packages/traitlets/config/configurable.py", line 405, in instance
    inst = cls(_args, *_kwargs)
  File "/Users/refops/bin/java9_kernel-master/javakernel/kernel.py", line 35, in **init**
    self._start_java_repl()
  File "/Users/refops/bin/java9_kernel-master/javakernel/kernel.py", line 54, in _start_java_repl
    continuation_prompt=u'>>'
  File "//anaconda/lib/python3.5/site-packages/pexpect/replwrap.py", line 55, in __init__
    self._expect_prompt()
  File "//anaconda/lib/python3.5/site-packages/pexpect/replwrap.py", line 66, in _expect_prompt
    timeout=timeout)
  File "//anaconda/lib/python3.5/site-packages/pexpect/spawnbase.py", line 384, in expect_exact
    return exp.expect_loop(timeout)
  File "//anaconda/lib/python3.5/site-packages/pexpect/expect.py", line 104, in expect_loop
    return self.timeout(e)
  File "//anaconda/lib/python3.5/site-packages/pexpect/expect.py", line 68, in timeout
    raise TIMEOUT(msg)
pexpect.exceptions.TIMEOUT: Timeout exceeded.
<pexpect.pty_spawn.spawn object at 0x103c5d7b8>
command: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/bin/java
args: ['/Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/bin/java', '-cp', '/Users/refops/bin/rheem-0.1-with-spark-1.6.0.SNAPSHOT.jar:/Users/refops/bin/kulla--20160615005912.jar', 'jdk.internal.jshell.tool.JShellTool']
searcher: None
buffer (last 100 chars): '|  Welcome to JShell -- Version 9-ea\r\n|  For an introduction type: /help intro\r\n\r\n\r\njshell> '
before (last 100 chars): '|  Welcome to JShell -- Version 9-ea\r\n|  For an introduction type: /help intro\r\n\r\n\r\njshell> '
after: <class 'pexpect.exceptions.TIMEOUT'>
match: None
match_index: None
exitstatus: None
flag_eof: False
pid: 26823
child_fd: 46
closed: False
timeout: 30
delimiter: <class 'pexpect.exceptions.EOF'>
logfile: None
logfile_read: None
logfile_send: None
maxread: 2000
ignorecase: False
searchwindowsize: None
delaybeforesend: 0.05
delayafterclose: 0.1
delayafterterminate: 0.1
ERROR: Kernel did not respond

Shutting down kernel

**Comment by Bachmann1234** [comment]
Looks like the same issue as https://github.com/Bachmann1234/java9_kernel/issues/17 we never did figure out what was going on.

I'll take another stab at this sometime this week.

**Comment by Bachmann1234** [comment]
Hey, I managed to fix this but I am hitting another issue. 

Basically the underlying kulla.jar had some changes that broke the kernel. I tried to update it. 

The kernel should now load but multiline code does not seem to be working despite my efforts :-/

I will open a new issue for that

**Comment by khayyatzy** [comment]
Thank you for your help
